presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestDoubleMinNAggregation.java

```diff
@@ -68,6 +70,14 @@ public void testMoreCornerCases()
         testCustomAggregation(new Double[] {1.0, 2.0, null, 3.0}, 5);
         testInvalidAggregation(new Double[] {1.0, 2.0, 3.0}, 0);
         testInvalidAggregation(new Double[] {1.0, 2.0, 3.0}, -1);
+
+        // Test min(val, n), to make sure n is consistent.
+        Block valueBlock = createDoublesBlock(new Double[] {1.0, 2.0, 3.0});
+        // There is an inconsistent N: 3 there.
+        Block nBlock = createLongsBlock(new Long[] {2L, 2L, 3L});
+        assertThrows(PrestoException.class, () -> {
```

Comment:
This returns what threw--could you also do a check on the error message to make sure it indicates what the values were?

Reply:
done